### PR TITLE
ash/DOCU-1683 [AWS PrivateLink checkbox - docs]

### PR DESCRIPTION
**Pull request by AshVic**
The Amazon S3 docs for configuring it as a source and for configuring it as a target both require proper procedural documentation which falls outside of the scope of this ticket.

I have added the necessary info for the checkbox, however, in the docs improvement sprint next sprint, the source and target procedural, step-by-step info need documentation in their own pages.

### CODE REVIEW

File: docs/configure-source.md [hunk 1]
```diff
@@ -136,17 +136,18 @@ To configure an Amazon S3 bucket as a source for LiveData Migrator, enter the fo
...
-  	 b) Add the source and specify a queue but no endpoint. This allows you to use a queue that exists in a public endpoint.
-       If you define your own queue, the queue must be attached to the S3 bucket. For more information about adding queues to buckets, view the [AWS documentation](https://docs.aws.amazon.com/AmazonS3/latest/userguide/grant-destinations-permissions-to-s3.html#grant-sns-sqs-permission-for-s3).
-  	 c) Add the source and specify a queue and a service endpoint. The endpoint can be a public or a private endpoint. For more information about public endpoints, view the [Amazon SQS endpoints documentation](https://docs.aws.amazon.com/general/latest/gr/sqs-service.html).
+   * **Use AWS PrivateLink** (Optional) - Select this checkbox if you want private connectivity between Amazon Simple Storage Service (S3) and on-premises resources using private IPs. See [AWS PrivateLink for Amazon S3](https://docs.aws.amazon.com/AmazonS3/latest/userguide/privatelink-interface-endpoints.html). After you select this checkbox, enter the endpoint that LiveData Migrator should use to connect to Amazon S3 in the **S3 Endpoint** field.
```
Comment (lucasdigital) [inline docs/configure-source.md:139]: On the daily build, this Use AWS PrivateLink checkbox-enabled field is labeled "**S3 VPC Endpoint**". Will this change to match the documentation? (Amazon S3 Target Filesystem Configuration) 10.6.1-SNAPSHOT.

File: docs/configure-target.md [hunk 2]
```diff
@@ -16,7 +16,8 @@ To add a target filesystem from your LiveData Migrator dashboard, select the fol
    * Azure Data Lake Storage (ADLS) Gen2 (preview) - The configuration depends on the **Authentication Type**:
      * [Shared Key](./command-reference.md#mandatory-parameters-2)
      * [Service Principal (OAuth2)](./command-reference.md#mandatory-parameters-1)
-   * [Amazon S3/IBM Cloud Object Storage (preview)](./command-reference.md#s3a-mandatory-parameters)
+   * [Amazon S3/IBM Cloud Object Storage (preview)](./command-reference.md#s3a-mandatory-parameters)  
+     **Use AWS PrivateLink** (Optional) - Select this checkbox if you want private connectivity between Amazon Simple Storage Service (S3) and on-premises resources using private IPs. See [AWS PrivateLink for Amazon S3](https://docs.aws.amazon.com/AmazonS3/latest/userguide/privatelink-interface-endpoints.html). After you select this checkbox, enter the endpoint that LiveData Migrator should use to connect to Amazon S3 in the **S3 Endpoint** field.
```
Comment (lucasdigital) [inline docs/configure-target.md:20]: As per previous comment. There is a possible inconsistency in how the Endpoint entry field is labeled.

**Review by lucasdigital — APPROVED** [review]
* Content appears correct
* Style and tone consistent with MS guidelines.
One small thing to check: the "S3 Endpoint" label for the entry field may be slightly different. I've attached a screenshot to DOCU-1683.

File: docs/configure-source.md (hunk 1 above)
Comment (AshVic) [reply docs/configure-source.md:139]: Yes the UI texts are changing

**Pull request merged**